website_sale_require_legal/static/tests/tours/tour.js

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define("website_sale_require_legal.tour", function (require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+
+    var steps = [
+        {
+            trigger: '.oe_product_cart a:contains("Customizable")',
+        },
+        {
+            trigger: "#add_to_cart",
+        },
+        {
+            trigger: "a[href='/shop/cart']",
+            extra_trigger: "sup.my_cart_quantity:contains('1')",
+        },
+        {
+            trigger: ".btn:contains('Process Checkout')",
+        },
+        {
+            trigger: "a[href='/shop/checkout']",
```

Comment:
```suggestion
            trigger: ".progress-wizard-steplabel:contains('Address')",
```

Try to correct the test by changing this step to this other

Reply:
The error occurs again at the same step

Reply:
So I'm not seeing the same error... Last error was related to this step https://github.com/OCA/e-commerce/pull/733/files#diff-23db7de503e2d4676b7fe9f495e9cdc5f4ed0e0304b51b68f6407766dc4a2062R27 now it is related to this other https://github.com/OCA/e-commerce/pull/733/files#diff-23db7de503e2d4676b7fe9f495e9cdc5f4ed0e0304b51b68f6407766dc4a2062R56

Reply:
You are right, the previous error was repeated because I had changed the code in the wrong step, when I have done it in the corresponding step the error is different, however it is a different error than the one I have in the local installation.